service/src/registry.ts

```diff
@@ -523,6 +558,53 @@ export class Registry {
     });
   }
 
+  /**
+   * @method            module:azure-iothub.Registry#updateModuleTwin
+   * @description         Updates the Twin of a specific module with the given patch.
+   *
+   * @param {String}      deviceId    The device identifier.
+   * @param {String}      moduleId    The module identifier
+   * @param {Object}      patch       The desired properties and tags to patch the module twin with.
+   * @param {string}      etag        The latest etag for this module twin or '*' to force an update even if
+   *                                  the module twin has been updated since the etag was obtained.
```

Comment:
I know that we have version update support.  Should we be adding this in here now.  Might want to talk with Pierre abou this.

Reply:
Are you talking about reported/desired properties version updates? that's on the client side isn't it?

---
In reply to: [176868854](https://github.com/Azure/azure-iot-sdk-node/pull/256#discussion_r176868854) [](ancestors = 176868854)